extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -34,6 +34,8 @@ export enum State {
 	EXIT,
 }
 
+export const BusinessCriticalTier = 'BusinessCritical';
```

Comment:
Should we turn this into an enum and also include GeneralPurpose while we're at it?

Reply:
good idea